packages/shorebird_cli/lib/src/shorebird_flutter.dart

```diff
@@ -218,4 +207,36 @@ class ShorebirdFlutter {
     shorebirdEnv.flutterRevision = revision;
     useFlutterProgress.complete();
   }
+
+  /// Helper method to return the human readable version from the
+  /// current flutter revision in [ShorebirdEnv.flutterRevision].
+  Future<String> humanReadableVersionFromEnv() async {
+    return humanReadableVersion(
+      flutterRevision: shorebirdEnv.flutterRevision,
+    );
+  }
+
+  /// Returns an human readable string from the received [flutterRevision] and
+  /// [flutterVersion].
+  ///
+  /// e.g. 3.16.3 (b9b2390296)
+  ///
+  /// If [flutterVersion] is null, it will be calculated from the current
+  ///   revision using [getVersionString].
+  Future<String> humanReadableVersion({
```

Comment:
Minor but looks like we renamed `getVersionAndRevision` to `humanReadableVersion`. I personally think the previous name was more descriptive of what this method does 🤷